pkg/apis/kops/cluster.go

```diff
@@ -205,6 +205,17 @@ type ClusterSpec struct {
 	RollingUpdate *RollingUpdate `json:"rollingUpdate,omitempty"`
 	// ClusterAutoscaler defines the cluster autoscaler configuration.
 	ClusterAutoscaler *ClusterAutoscalerConfig `json:"clusterAutoscaler,omitempty"`
+	// IAMRolesForServiceAccount defines the IRSA configuration.
+	IAMRolesForServiceAccounts *IAMRolesForServiceAccountsConfig `json:"iamRolesForServiceAccounts,omitempty"`
```

Comment:
I'm wondering if we should merge these fields into IAMSpec or if they're worth keeping separate.

Reply:
I think it should be its own struct. If not, we'll have to prefix all the fields. The entire struct could be moved to IAMSpec, I suppose. IAMSpec also doesn't really have "a clear purpose".